src/Resources/contao/drivers/DC_Table.php

```diff
@@ -3433,11 +3433,11 @@ protected function treeView()
 			if (!is_array($session[$node]) || empty($session[$node]) || current($session[$node]) != 1)
 			{
 				$session[$node] = array();
-				$objNodes = $this->Database->execute("SELECT DISTINCT pid FROM " . $table . " WHERE pid>0");
+				$objNodes = $this->Database->execute("SELECT DISTINCT id FROM " . $table);
```

Comment:
This cannot be right. The original statement says "select all PIDs greater 0" which is **not the same** as "select all IDs". Consider this:

```
------------
| id | pid |
------------
|  1 |   0 |
|  2 |   1 |
------------
```

The original query would return `1` and your query would return `1, 2`.

Reply:
I changed the node toggle method, too. So in this context it's right.

In the old way, only pages with subpages could be toggled. The child table rows (e.g. articles) of a page are always visible. 
With this PR I changed it and all pages with subpages or child table rows can be toggled. 
That's why I need all page ids here.